### DISCUSSION TITLE
Shading with multiple surface layers

### DISCUSSION THREAD

**Discussion by innuo** [tylermorganwall/rayshader#194] [Q&A]
Hi,

First, thanks for your fantastic R package. 

I am trying to visualize the amount of sunlight falling on a solar array at various tilts of the solar array table. It would be nice to have the solar panels shown as a being perched on top of a vertical wall or hovering in space (rather than as rectangular prisms).

It's not possible to specify this setup with a height matrix, because of the empty space between the panels and the ground. Is there a way to accomplish this?

**Comment by tylermorganwall** [comment]
No current way to do this.
